dashboard/app/models/pd/workshop.rb

```diff
@@ -197,7 +197,7 @@ class Pd::Workshop < ActiveRecord::Base
   before_save :process_location, if: -> {location_address_changed?}
   auto_strip_attributes :location_name, :location_address
 
-  before_save :assign_regional_partner, if: -> {organizer_id_changed?}
+  before_save :assign_regional_partner, if: -> {organizer_id_changed? && !regional_partner_id?}
```

Comment:
Also redundant with that change - but lets see which gets merged first

Reply:
The race is on...